src/Classes/Utils/EventListeners.ts

```diff
@@ -45,7 +47,15 @@ export class EventListeners {
     /**
      * Initializing event listeners
      */
-    public addEventsListeners(): void {
+    public addListeners(): void {
+        this.manageListeners(true);
+    }
+
+    public removeListeners(): void {
+        this.manageListeners(false);
+    }
+
+    private manageListeners(add: boolean) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'manageListeners' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)